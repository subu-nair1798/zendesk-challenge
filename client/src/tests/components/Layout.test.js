import React from 'react';
import { shallow } from 'enzyme';
import Layout from '../../components/Layout';

describe('<Layout />', () => {
    it('renders Page Layout', () => {
        const wrapper = shallow(<Layout />);
        expect(wrapper).toMatchSnapshot();
    });
});