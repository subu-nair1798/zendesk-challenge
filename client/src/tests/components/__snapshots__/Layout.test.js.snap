// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> renders Page Layout 1`] = `
<Container>
  <Header />
</Container>
`;
